homeassistant/components/insteon/config_flow.py

```diff
@@ -7,14 +7,16 @@
 
 from pyinsteon import async_connect
 
-from homeassistant.components import dhcp, usb
+from homeassistant.components import usb
```

Comment:
Note: usb is still imported
It is listed as `after_dependencies` in manifest.json